### ISSUE TITLE
`emamux:send-command` with argument

### ISSUE THREAD

**Issue by sriramkswamy** [syohex/emacs-emamux#45]
I want to programmatically send commands to a tmux pane that is not necessarily invoked by `emamux:run-command`. Is there any way to do this? `emamux:send-command` doesn't accept arguments.

**Comment by syohex** [comment]
Sorry late response.

`emamux:set-parameters` queries target session if it is not decided yet. So I think `target` should be also passed as argument. How about following patch ?

```diff
diff --git a/emamux.el b/emamux.el
index 7baab89..08e768b 100644
--- a/emamux.el
+++ b/emamux.el
@@ -224,17 +224,17 @@ match \"buffer[0-9]+\" in its first subexp as well."
     (error "'tmux' does not run on this machine!!")))
 
 ;;;###autoload
-(defun emamux:send-command ()
+(defun emamux:send-command (&optional command target)
   "Send command to target-session of tmux"
   (interactive)
   (emamux:check-tmux-running)
   (condition-case nil
       (progn
-        (if (or current-prefix-arg (not (emamux:set-parameters-p)))
-            (emamux:set-parameters))
+        (when (and (not target) (or current-prefix-arg (not (emamux:set-parameters-p))))
+          (emamux:set-parameters))
         (let* ((target (emamux:target-session))
                (prompt (format "Command [Send to (%s)]: " target))
-               (input  (emamux:read-command prompt t)))
+               (input  (or command (emamux:read-command prompt t))))
           (emamux:reset-prompt target)
           (emamux:send-keys input)))
       (quit (emamux:unset-parameters))))
```

**Comment by sriramkswamy** [comment]
Yes, this works perfectly when I tried locally. Will this be available upstream soon?

**Comment by syohex** [comment]
I have merged it at #47
